packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -32,19 +32,37 @@ import { useBorderPanelLabel } from '../../hooks/border';
 
 import { unlock } from '../../lock-unlock';
 
+function BlockStylesPanel( { clientId } ) {
+	return (
+		<div>
+			<PanelBody title={ __( 'Styles' ) }>
+				<BlockStyles clientId={ clientId } />
+			</PanelBody>
+		</div>
```

Comment:
Any idea why we need wrapper `<div>` here?

Reply:
No, but tested now and it seems it doesn't affect something. I'll remove it.